Sources/swift-zenz-coreml/swift_zenz_coreml.swift

```diff
@@ -3,68 +3,117 @@
 import CoreML
 import Foundation
 
-func main() {
-    // カスタムクラスの定義
-    class ModelInput: MLFeatureProvider {
-        var inputIds: MLMultiArray
-        var attentionMask: MLMultiArray
-
-        init(inputIds: MLMultiArray, attentionMask: MLMultiArray) {
-            self.inputIds = inputIds
-            self.attentionMask = attentionMask
-        }
+// Tokenizer 구조체 정의
+// Define the Tokenizer struct
+struct Tokenizer {
+    let vocab: [String: Int]
+    let reverseVocab: [Int: String]
+    let config: [String: Any]
 
-        var featureNames: Set<String> {
-            return ["input_ids", "attention_mask"]
-        }
-
-        func featureValue(for featureName: String) -> MLFeatureValue? {
-            if featureName == "input_ids" {
-                return MLFeatureValue(multiArray: inputIds)
-            }
-            if featureName == "attention_mask" {
-                return MLFeatureValue(multiArray: attentionMask)
-            }
-            return nil
-        }
+    // 초기화
+    // Initializer
+    init(vocabFile: String, configFile: String) {
+        // 번들에서 파일 URL 가져오기
+        // Get the file URLs from the bundle
+        let vocabURL = Bundle.main.url(forResource: vocabFile, withExtension: "json")!
+        let configURL = Bundle.main.url(forResource: configFile, withExtension: "json")!
```

Comment:
Please use `Bundle.module.url` instead of `Bundle.main.url` as it works better in Swift Package

Reply:
It look great! I fixed it [bfcca57](https://github.com/ensan-hcl/swift-zenz-coreml/pull/1/commits/bfcca5712f85632b914d99655e235493a8e34a31)